go-controller/pkg/ovn/pods.go

```diff
@@ -170,16 +172,36 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) {
 	portName := fmt.Sprintf("%s_%s", pod.Namespace, pod.Name)
 	logrus.Debugf("Creating logical port for %s on switch %s", portName, logicalSwitch)
 
-	out, err := exec.Command(OvnNbctl, "--wait=sb", "--", "--may-exist", "lsp-add",
-		logicalSwitch, portName, "--", "lsp-set-addresses",
-		portName, "dynamic", "--", "set",
-		"logical_switch_port", portName,
-		"external-ids:namespace="+pod.Namespace,
-		"external-ids:logical_switch="+logicalSwitch,
-		"external-ids:pod=true").CombinedOutput()
-	if err != nil {
-		logrus.Errorf("Error while creating logical port %s - %v (%s)", portName, err, string(out))
-		return
+	annotation, ok := pod.Annotations["ovn"]
```

Comment:
Since "ok" is being used at quite a few places in the function, I think it could use a more appropriate name. e.g: staticIPAddress

Reply:
Done